<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgChangePassword</class>
 <widget class="QDialog" name="DlgChangePassword">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>614</width>
    <height>281</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>13</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="lineEdit_newpassword">
   <property name="geometry">
    <rect>
     <x>115</x>
     <y>150</y>
     <width>113</width>
     <height>40</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>210</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>确认新密码:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_oldpssword">
   <property name="geometry">
    <rect>
     <x>115</x>
     <y>90</y>
     <width>113</width>
     <height>40</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_newpassword_2">
   <property name="geometry">
    <rect>
     <x>115</x>
     <y>210</y>
     <width>113</width>
     <height>40</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>85</y>
     <width>56</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>原密码:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>145</y>
     <width>56</width>
     <height>36</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>新密码:</string>
   </property>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget_keytype">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>231</width>
     <height>61</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="page_user">
    <widget class="QRadioButton" name="radioButton_midkey">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>20</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>中级</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_lowkey">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string> 初级</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_upkey">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>20</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>高级</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_operator">
    <widget class="QRadioButton" name="radioButton_developkey">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>20</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>开发</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_debugkey">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>调试</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QFrame" name="frame_1">
   <property name="geometry">
    <rect>
     <x>258</x>
     <y>30</y>
     <width>237</width>
     <height>231</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="btn_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>12</y>
      <width>64</width>
      <height>45</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sans Serif</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_8">
    <property name="geometry">
     <rect>
      <x>84</x>
      <y>12</y>
      <width>64</width>
      <height>45</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sans Serif</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_9">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>12</y>
      <width>64</width>
      <height>45</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sans Serif</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_6">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>68</y>
      <width>64</width>
      <height>45</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sans Serif</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_4">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>68</y>
      <width>64</width>
      <height>45</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sans Serif</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_5">
    <property name="geometry">
     <rect>
      <x>84</x>
      <y>68</y>
      <width>64</width>
      <height>45</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sans Serif</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_3">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>124</y>
      <width>64</width>
      <height>45</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sans Serif</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_2">
    <property name="geometry">
     <rect>
      <x>84</x>
      <y>124</y>
      <width>64</width>
      <height>45</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sans Serif</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_1">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>124</y>
      <width>64</width>
      <height>45</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sans Serif</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_right">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>180</y>
      <width>64</width>
      <height>45</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sans Serif</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>→</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_left">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>180</y>
      <width>64</width>
      <height>45</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sans Serif</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>←</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_0">
    <property name="geometry">
     <rect>
      <x>84</x>
      <y>180</y>
      <width>64</width>
      <height>45</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sans Serif</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frame_2">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>30</y>
     <width>91</width>
     <height>109</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="btn_clear">
    <property name="geometry">
     <rect>
      <x>8</x>
      <y>10</y>
      <width>71</width>
      <height>42</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>AR PL UKai TW</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>清除</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_backspace">
    <property name="geometry">
     <rect>
      <x>8</x>
      <y>60</y>
      <width>71</width>
      <height>42</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>AR PL UKai TW</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>退格</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="btn_ok">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>160</y>
     <width>91</width>
     <height>42</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AR PL UKai TW</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_back">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>210</y>
     <width>91</width>
     <height>42</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AR PL UKai TW</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
